varcode/frameshift_coding_effect.py

```diff
@@ -118,8 +118,84 @@ def _frameshift(
         transcript=transcript,
         aa_mutation_start_offset=mutation_start_position,
         aa_ref=aa_ref,
-        shifted_sequence=mutant_protein_suffix)
+        shifted_sequence=str(mutant_protein_suffix))
 
+def cdna_codon_sequence_after_insertion_frameshift(
+        sequence_from_start_codon,
+        cds_offset_before_insertion,
+        inserted_nucleotides):
+    """
+    Returns index of mutated codon and nucleotide sequence starting at the first
+    mutated codon.
+    """
+    # special logic for insertions
+    coding_sequence_after_insertion = \
+        sequence_from_start_codon[cds_offset_before_insertion + 1:]
+
+    if cds_offset_before_insertion % 3 == 2:
+        # insertion happens after last nucleotide in a codon,
+        # doesn't disrupt the existing codon from cds_offset-2 to cds_offset
+        mutated_codon_index = cds_offset_before_insertion // 3 + 1
+        nucleotides_before = ""
+    elif cds_offset_before_insertion % 3 == 1:
+        # insertion happens after 2nd nucleotide of a codon
+        # codon positions:
+        #   1) cds_offset - 1
+        #   2) cds_offset
+        #    <----- Insertsion
+        #   3) cds_offset + 1
+        mutated_codon_index = cds_offset_before_insertion // 3
+        nucleotides_before = sequence_from_start_codon[
```

Comment:
Add comment `The first two reference nucleotides before the insertion + the insertion result in the first mutant codon`?